### ISSUE TITLE
Icon and Text not visible on widget action pane when specified in configuration.yaml

### ISSUE THREAD

**Issue by BUSTERCJB** [home-assistant/iOS#1680]
iOS device model, version and app version

Model Name: iPhone 12 Pro
Software Version: iOS 14.6
App version: 2021.6 (2021.158)

Home Assistant Core Version - 2021.6.5

Describe the bug

When I programmed the action in configuration.yaml as per https://companion.home-assistant.io/docs/core/actions/, I found that the icon and the text would not show up on the iPhone for the action on the widget pane, nor the app configuration pane in the actions section on the iPhone. Only the background colour of the widget would be reflected on the widget. 

To Reproduce
I restarted home assistant and the app numerous times and I couldn't get it to show an icon nor text on the widget. The same thing occurred when I tried using the Mac companion app as well, so I assume the issue is shared.

Expected Behaviour
I would assume that an icon and a widget would be shown, as is the case when the action/widget is directly programmed onto the iPhone in the app configuration pane (therefore, not by configuration into configuration.yaml).

Screenshots
![Screenshot 2021-06-21 at 10 21 49 pm](https://user-images.githubusercontent.com/40330747/122761180-4c070300-d2df-11eb-9aaa-3c852b72f4af.jpeg)
![Screenshot 2021-06-21 at 10 22 24 pm](https://user-images.githubusercontent.com/40330747/122761200-50cbb700-d2df-11eb-8cf6-a16747f2ef80.jpeg)
![Screenshot 2021-06-21 at 10 22 50 pm](https://user-images.githubusercontent.com/40330747/122761218-545f3e00-d2df-11eb-9a5d-b7f85fffe587.jpeg)

Additional context
"
ios:
  actions:
    - name: Front Gate
      background_color: "#0099ff"
      label:
        text: "Front Gate"
        color: "##000000"
      icon:
        icon: gate
        color: "##000000"
 "

**Comment by TomBrien** [comment]
Can you please format your example `configuration.yaml` entry as code and check the indents but I notice that the `color` values under both `icon:` and `text:`are invalid hex codes. Both have double `#` at the start instead of a single. I'm not sure we except `#0`for black. Try replacing both with `#000000` and reloading the iOS integration or restart home assistant.

**Comment by BUSTERCJB** [comment]
Sorry, I coped and pasted in the config code and didn't bother checking if I copied it with correct formatting. Somehow the ##000000 went to #0 without me noticing when pasting it in.  You were right, it was the double `#`. Don't know how I didn't notice that one. Must've copied in the colour code from a website and it resulted in double `#`. Thank you for spotting my error. 

`ios:
  actions:
    - name: Front Gate
      background_color: "#0099ff"
      label:
        text: "Front Gate"
        color: "#ffffff"
      icon:
        icon: gate
        color: "#ffffff"`
